log_outgoing_requests/handlers.py

```diff
@@ -4,41 +4,70 @@
 
 from django.conf import settings
 
+ALLOWED_CONTENT_TYPES = [
+    "application/json",
+    "multipart/form-data",
+    "text/html",
+    "text/plain",
+    "",
+    None,
+]
```

Comment:
as remarked above, this should be a configurable setting

`multipart/form-data` should also not be in this list by default, since that's used for file uploads too. That should be explicitly opt-in by library users.

Missing content type information should also be treated as potentially dangerous and by default NOT be logged. Please always ensure the "safe by default" principles.